pyproject.toml

```diff
@@ -175,10 +175,8 @@ max-statements = 70
 "openlibrary/plugins/upstream/borrow.py" = ["BLE001", "E722"]
 "openlibrary/plugins/upstream/models.py" = ["BLE001"]
 "openlibrary/plugins/upstream/utils.py" = ["BLE001"]
-"openlibrary/solr/*" = ["UP007"]
+"openlibrary/solr/solr_types.py" = ["UP007"]
```

Comment:
What stops us from migrating `openlibrary/solr/solr_types.py` as well?

Reply:
Nothing ! That file is auto-generated though, so we'd need to change the file that generates it ( `types_generator.py` )

Reply:
And the two `# noqa: UP007` in `scripts/solr_builder/tests/test_fn_to_cli.py`